src/devices/shelly.ts

```diff
@@ -1,33 +1,233 @@
+import {Zcl} from "zigbee-herdsman";
 import * as m from "../lib/modernExtend";
 import type {DefinitionWithExtend} from "../lib/types";
+import {assertString} from "../lib/utils";
+
```

Comment:
Instead of the two functions, define a `const modernExtendLocal` here like e.g. https://github.com/Koenkk/zigbee-herdsman-converters/blob/6c45dbce612a537b040ff327a0b4d3a1c647e46a/src/lib/tuya.ts#L2100 and use it in the definitions like `modernExtendLocal. shellyRPCCluster()`

Reply:
This is fine, thanks!